ui/snippets/walletMenu/WalletIdenticon.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Flex, chakra } from '@chakra-ui/react';
+import React from 'react';
+
+import useIsMobile from 'lib/hooks/useIsMobile';
+import AddressIdenticon from 'ui/shared/entities/address/AddressIdenticon';
+import IconSvg from 'ui/shared/IconSvg';
+
+import useMenuButtonColors from '../useMenuButtonColors';
+
+type Props = {
+  address: string;
+  isAutoConnectDisabled?: boolean;
+  className?: string;
+};
+
+const WalletIdenticon = ({ address, isAutoConnectDisabled, className }: Props) => {
+  const { themedBackgroundOrange } = useMenuButtonColors();
+  const isMobile = useIsMobile();
+
+  return (
+    <Box className={ className } position="relative">
+      <AddressIdenticon size={ 20 } hash={ address }/>
+      { isAutoConnectDisabled && (
```

Comment:
Please consider adding a test to cover this functionality.

The same comment about the new code in `ui/snippets/walletMenu/WalletMenuContent.tsx`

Reply:
Agreed to do it in a separate task